olp/utils.py

```diff
@@ -105,6 +133,7 @@ def patch_models():
 
     setattr(User, "assign_perm", assign_perm)
     setattr(User, "remove_perm", remove_perm)
+    setattr(User, "remove_all_permissions", remove_all_permissions)
```

Comment:
Ok, I wasn't sure on that. I was going for consistency with existing methods but I've reverted the change.